src/main.rs

```diff
@@ -191,7 +192,8 @@ fn main() -> Result<(), lexopt::Error> {
     }
 
     let config = {
-        if let Ok(mut f) = File::options().read(true).open(CONFIG_FILE_NAME) {
+        let open_config = |file_name| File::options().read(true).open(file_name);
+        if let Ok(mut f) = open_config(CONFIG_FILE_NAME).or(open_config(DOT_CONFIG_FILE_NAME)) {
```

Comment:
Aaaah, it just clicked for me. Here is where I need to inline it. Got it.

Reply:
tell me when you're ready for review ;)